server/portal/apps/system_monitor/views.py

```diff
@@ -1,22 +1,19 @@
 from django.conf import settings
-from datetime import datetime, timedelta
 from portal.views.base import BaseApiView
 from django.http import JsonResponse
-import dateutil.parser
 import requests
 import json
 import logging
-import pytz
 
 logger = logging.getLogger(__name__)
 
 
-def _get_unoperational_system(hostname):
-    return {'hostname': hostname,
-            'display_name': hostname.split('.')[0].capitalize(),
+def _get_unoperational_system(display_name):
+    return {'display_name': display_name,
             'is_operational': False,
-            'load_percentage': 0,
-            'jobs': {'running': 0, 'queued': 0, 'other': 0},
+            'load': 0,
+            'running': 0,
+            'waiting': 0,
```

Comment:
For now, let's keep the `jobs` object, as there are more side effects in the system monitor snippet when refactoring. We can refactor in a follow up task.

Reply:
Seems we don't use `other`, so could leave that out: `'jobs': {'running': 0, 'queued': 0},`